packages/dev-server/src/dev-server.ts

```diff
@@ -20,7 +20,7 @@ export type DevServerOptions = {
 export const defaultOptions: Required<Omit<DevServerOptions, 'cf'>> = {
   entry: './src/index.ts',
   injectClientScript: true,
-  exclude: ['.*.ts', '.*.tsx', '/@.+', '/node_modules/.*'],
+  exclude: ['.*.ts', '.*.tsx', '/@.+', '/node_modules/.*', '/inc/.*', '.*.txt', '.*.ico' ],
```

Comment:
```suggestion
  exclude: ['.*\\.ts', '.*\\.tsx', '/@.+', '\\/node_modules\\/.*', '\\/inc\\/.*', '.*\\.txt', '.*\\.ico' ],
```
I fixed the original default exclusions in https://github.com/honojs/vite-plugins/pull/9 , but these new exclusions should be fixed too